test/e2e/manifests/mixins/prometheus-deployment.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prometheus
+spec:
+  replicas: 0
```

Comment:
if this is only available in `PROFILE=prometheus` I think you don't need this because you want to be `replicas=1`